src/KygekDev/Height/Height.php

```diff
@@ -31,18 +31,15 @@ public function onCommand(CommandSender $sender, Command $command, string $label
         if ($command->getName() !== "height" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
 
         if (!isset($args[0])) {
-            if (!$sender instanceof Player) {
-                $this->sendMessage($sender, "Usage: /height <player>");
-                return true;
-            }
+            if (!$sender instanceof Player) return false;
 
             $height = $this->getHeight($sender);
             $this->sendMessage($sender, "You're " . ($height >= 0 ? $height . "m above" : abs($height) . "m below") . " sea level");
             return true;
         }
 
-        $player = $this->getServer()->getPlayer($args[0]);
-        if ($player === null) {
+        $player = $this->getServer()->getPlayerByPrefix($args[0]);
+        if (!$player) {
```

Comment:
If `$player` is `null`, will `!$player` return `true`?

Reply:
Exactly!

Reply:
`!$player` Faster

Reply:
https://3v4l.org/ARdc6

Reply:
So... Does it need re-editing?

Reply:
> So... Does it need re-editing?

No need